components/AddEventForm/index.js

```diff
@@ -12,14 +13,71 @@ export default function AddEventForm({ onSubmit, appointment }) {
     background-color: hsla(0, 0%, 4%, 0.64);
     color: lightgray;
     border-radius: 2rem;
+    padding: 0.5rem 1rem;
+    max-width: 540px;
+  `;
+  const StyledSelectSection = styled.section`
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin: 0.5rem auto;
+    border: 2px solid lightgray;
+    border-radius: 1rem;
   `;
   const StyledSubmitButton = styled.button`
-    width: 3rem;
+    width: 5rem;
+    margin: 0.5rem auto;
+    height: 5rem;
+    border-radius: 999px;
+    background-color: hsla(103, 100%, 34%, 0.89);
   `;
   const StyledParagraph = styled.p`
     align-self: center;
     font-size: 1.4rem;
   `;
+  const StyledLabel = styled.label`
+    font-size: 1.2rem;
+    margin 0.5rem auto
+  `;
+  const StyledSelect = styled.select`
+    width: 95%;
+    margin: 0.5 auto;
+    border-radius: 999px;
+    height: 2rem;
+    text-align: center;
+    background-color: lightgray;
+  `;
+  const StyledInput = styled.input`
+    width: 80%;
+    margin: 0 auto;
+    padding: 0.2rem;
+    height: 2rem;
+    font-size: 1.3rem;
+    border-radius: 999px;
+    background-color: lightgray;
+  `;
+  const StyledInputCost = styled.input`
+    width: 95%;
+    margin: 0.5rem auto;
+    border-radius: 999px;
+    height: 2rem;
+    text-align: center;
+    background-color: lightgray;
+  `;
+  const StyledDateInput = styled.input`
+    width: 95%;
+    margin: 0.5 auto;
+    border-radius: 999px;
+    height: 2rem;
+    text-align: center;
+    background-color: lightgray;
+  `;
```

Comment:
those inputs look almost identical. Is it possible to have only one styled component you use in both places?

Reply:
Done